js/private/js_run_devserver.bzl

```diff
@@ -4,103 +4,20 @@ load(":js_binary.bzl", "js_binary_lib")
 load(":js_binary_helpers.bzl", _gather_files_from_js_providers = "gather_files_from_js_providers")
 load("@bazel_skylib//lib:dicts.bzl", "dicts")
 
-_DOC = """Runs a devserver via binary target or command.
```

Comment:
Sadly not possible since the docs for a macro have to be an inline string and not a variable

Reply:
Could still do the macro in a separate commit but I guess that's its own mess then :(

Reply:
Not with the `tool` attribute change since I need to macro to pipe `tool` from the public API to two different attributes in the rule now that allow you to switch between `cfg = "exec"` and `cfg = "target"`